TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -456,6 +462,49 @@ class RouteDetailContentViewController: UIViewController {
 
     /// Draw all waypoints initially for all paths in [Path] or [[CLLocationCoordinate2D]], plus fill bounds
     private func drawMapRoute() {
+        var pathCount = 0
+        // Helper function to create bus stop circles
+        func busStopCircles(at coordinate: CLLocationCoordinate2D, on mapView: GMSMapView) -> GMSCircle {
+            let circle = GMSCircle(position: coordinate, radius: 50)
+            circle.fillColor = UIColor.white.withAlphaComponent(1.0)
+            circle.strokeColor = UIColor.black
+            circle.strokeWidth = 2.0
+            circle.map = mapView
+            circle.zIndex = 2
+            return circle
+        }
+        // Helper function to map final location marker
```

Comment:
put an newline above this line

Reply:
I originally had it that way but Xcode kept screaming at me about the "Trailing Whitespace Violation" 😔 but added it in now!

![image](https://github.com/user-attachments/assets/706f8001-2596-4d77-8c21-a5b4189a3f49)